packages/web/src/scss/components/Header/README.md

```diff
@@ -170,8 +170,12 @@ control here.
 </nav>
 ```
 
+<!--lint disable no-duplicate-headings -->
```

Comment:
I would be happier to change the titles and not add `lint disable`.

Reply:
Well, I have been thinking a while about this and I have no idea, how to do it better. Feel free to suggest a change.

Reply:
How about adding the 3rd level title before? eg. somehting like `Desktop-Only Actions Navigation`?

Reply:
Ok, updated both places. Thanks.